hknweb/candidate/views.py

```diff
@@ -10,8 +10,10 @@
 from django.contrib.staticfiles.finders import find
 from django.contrib.staticfiles.templatetags.staticfiles import static
 from random import randint
+import datetime
 
 from .models import OffChallenge, Announcement
+from ..events.models import Event
```

Comment:
Why the double dot?

Reply:
Oh, it’s from the events in the parent folder...
Uh, looks a little weird but I guess it works.